### PR TITLE
feat(example): add screen interaction buttons

### PR DESCRIPTION
**Pull request by xanpj**
Who knew there where keyboard bindings in this demo? There are! So I added a button to display them and a fullscreen button, useful when working on large digital whiteboards.

Screen Shot 2020-06-04 at 11 55 45 AM

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
This is awesome, thanks @xanpj !

Can you maybe fix the [linting errors](https://travis-ci.com/github/pinussilvestrus/postit-js/builds/169563840) before? Then It would be ready to merge

**Comment by pinussilvestrus** [comment]
I think executing `npm run lint -- --fix` will do most of the trick.

File: example/app/css/app.css [hunk 1]
```diff
@@ -4,6 +4,138 @@ html, body, #canvas {
...
+@font-face {font-family: "Font Awesome 5 Free Solid";
+  src: url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.eot"); /* IE9*/
+  src: url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.eot?#iefix") format("embedded-opentype"), /* IE6-IE8 */
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.woff2") format("woff2"), /* chrome、firefox */
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.woff") format("woff"), /* chrome、firefox */
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.ttf") format("truetype"), /* chrome、firefox、opera、Safari, Android, iOS 4.2+*/
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.svg#Font Awesome 5 Free Solid") format("svg"); /* iOS 4.1- */
+}
+
+.io-editing-tools {
```
Comment (pinussilvestrus) [inline example/app/css/app.css:22]: I think we should align the button styles with the existing ones:

image

cf. https://github.com/pinussilvestrus/postit-js/blob/master/example/app/css/app.css#L26

Comment (xanpj) [reply example/app/css/app.css:22]: Great idea! By align, do you mean style (shadow, height?) or pull the new buttons down?

Comment (pinussilvestrus) [reply example/app/css/app.css:22]: yep, exactly

**Comment by xanpj** [comment]
Screen Shot 2020-06-04 at 1 49 38 PM

Done! I left the height of the bottoms to align with the palette bottoms on the left, rather then the height of the bottom buttons. Also, the horizontal scrolling does not work currently, so I changed the keyboard map.

**Comment by pinussilvestrus** [comment]
Thanks a lot!

**Pull request merged**